docs/src/fordevelopers/quickconnect.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Intoruction to Developers.
+helpfulVotes: false
+---
+
+# Quick Connect
+
+``` List
+SYNOPSIS
+Quickly connect your app or client to Imversed services
+
+```
+
+## Public Available Endpoints
+
+Below is a list of publicly available endpoints that you can use to connect to the Imversed mainnet and public testnets:
+
+::: tip
+
+You can also use [chainlist.org](https://chainlist.org/) to add the node directly to [Metamask](/forusers/digitalwallets.md).
+
+:::
+
+:::: tabs ::: tab Mainnet
+
+|                     Address                    | Category | Maintainer |
+|----------------------------------------------- |----------|------------|
+|`https://jsonrpc-endpoint-test.imversed.com:443`|          |            |                
+|`https://query-endpoint-test.imversed.com:443`  |          |            |                
+|`https://tx-endpoint-test.imversed.com:443`     |          |            |                
+|`https://grpc-endpoint-test.imversed.com:443`   |          |            |                
+|`https://query-endpoint-canary.imversed.com:443`|          |            |                
+|`https://tx-endpoint-canary.imversed.com:443`   |          |            |                
+|`https://grpc-endpoint-canary.imversed.com:443` |          |            |                
+             
+
+::: ::: tab Testnet
+
+|                     Address                    | Category | Maintainer |
+|----------------------------------------------- |----------|------------|
+|`https://jsonrpc-endpoint-test.imversed.com:443`|          |            |                
+|`https://query-endpoint-test.imversed.com:443`  |          |            |                
+|`https://tx-endpoint-test.imversed.com:443`     |          |            |                
+|`https://grpc-endpoint-test.imversed.com:443`   |          |            |                
+|`https://query-endpoint-canary.imversed.com:443`|          |            |                
+|`https://tx-endpoint-canary.imversed.com:443`   |          |            |                
+|`https://grpc-endpoint-canary.imversed.com:443` |          |            |    
```

Comment:
Update links and endpoints for our chains. Here are new.

TESTNET
https://txe-test.imversed.network/ // transactions-explorer

https://rpc-test.imversed.network:443
https://grpc-test.imversed.network:443
https://rest-test.imversed.network:443
https://jsonrpc-test.imversed.network:443
https://ws-jsonrpc-test.imversed.network:443

Reply:
Updated.